prosessering-core/src/main/kotlin/no/nav/familie/prosessering/domene/Status.kt

```diff
@@ -1,7 +1,7 @@
 package no.nav.familie.prosessering.domene
 
 enum class Status {
-    FERDIG, FEILET, PLUKKET, BEHANDLER, KLAR_TIL_PLUKK, UBEHANDLET, AVVIKSHÅNDTERT, MANUELL_OPPFØLGING;
+    FERDIG, FEILET, PLUKKET, BEHANDLER, KLAR_TIL_PLUKK, UBEHANDLET, AVVIKSHÅNDTERT, MANUELL_OPPFØLGING, FERDIG_NÅ_FEILET_FØR;
```

Comment:
Hmm, litt usikker på denne. Dette er en status på tasken som ikke burde trenge en slik status tenker jeg. Tenker også at det er `TaskWorker` som burde sette status på tasker, som gjøres i `doActualWork`
Hvis man eks skulle sette `FERDIG_NÅ_FEILET_FØR` på en task i en task(`AsyncTaskStep`), så skriver taskworker over den sen. 

Kan man i stedet lage en query som henter tasks som har status ferdig, men som har en feilet i tasklogg?

Reply:
Jeg synes det er en bedre løsning. Synes det er unødvendig med en ekstra FERDIG status

Reply:
Blant annet med ny status så må man huske å oppdatere opprydding av FERDIG tasker til å inkludere ekstra status, og evt andre funksjoner som baserer seg på FERDIG

Reply:
Her ser eg at koden min var utydeleg, for det var aldri tanken at denne statusen skulle setjast på noko eller brukast til anna enn søket.

Men det å lage ein ny query høyrast ut som ei betre løysing. Skal sjå på det.